proposals/2025-06-23-tsc-voting-procedures.md

```diff
@@ -0,0 +1,177 @@
+# TSC Voting Procedures Proposal
+
+## Summary
+
+This proposal establishes standardized voting procedures for the Hiero Technical Steering Committee (TSC) to improve decision-making efficiency while maintaining transparency and adherence to charter requirements.
+
+## Motivation
+
+The TSC frequently struggles to achieve quorum during scheduled meetings, resulting in delayed decision-making. A standardized asynchronous voting process via GitHub Pull Requests will enable the TSC to maintain momentum while ensuring all members can participate regardless of meeting attendance.
```

Comment:
The TSC frequently struggles to achieve quorum during scheduled meetings, resulting in delayed decision-making. <- It happened from time to time but I would not define thata as the main motivation.